### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by marco-m-pix4d** [dave/courtney#29]
Hello @dave,

just wanted to thank you for courtney, greatly appreciated!

marco

**Comment by dave** [comment]
No problem 😊

**Comment by randomvariable** [comment]
Yes, thank you for this. May your life be filled with flat nosed geezers.
